autospec/buildreq.py

```diff
@@ -632,6 +632,9 @@ def scan_for_configure(dirn):
         add_buildreq("buildreq-scons")
     elif buildpattern.default_pattern == "R":
         add_buildreq("buildreq-R")
+    elif buildpattern.default_pattern == "php":
+        add_buildreq("php")
```

Comment:
Do you want to add a buildreq-php package for these?

Reply:
I think so, because I currently see that it already does need:
```
php
php-dev
re2c
```